components/ee/agent-smith/pkg/network/egress.go

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+// Licensed under the Gitpod Enterprise Source Code License,
+// See License.enterprise.txt in the project root folder.
+
+package network
```

Comment:
At this point I'm not convienced that this single, quite short functuion, warants its own package

Reply:
I agree

Reply:
Done